sw/device/tests/BUILD

```diff
@@ -1868,3 +1868,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
```

Comment:
This should be moved to `sw/device/tests/sim_dv/BUILD`

Reply:
see my note above.